components/contributors/InfoCard.tsx

```diff
@@ -38,15 +38,22 @@ export default function InfoCard({
             />
           </div>
         </Link>
-        <div className={minimal ? "" : "flex flex-col items-center space-y-2"}>
-          <div className="font-medium text-lg space-y-1">
+        <div
+          className={`overflow-hidden ${
+            minimal
+              ? ""
+              : "flex flex-col items-center space-y-2 overflow-hidden"
+          }
+          `}
+        >
```

Comment:
We are having "overflow-hidden" class applied multiple times if `minimal=true`

Reply:
Sorry, mistake. Changed